test/base/TimeSpanTest.ooc

```diff
@@ -128,6 +128,13 @@ TimeSpanTest: class extends Fixture {
 			expect(Bstring == "0 0 text 0 60")
 			(defaultString, dayString, Astring, Bstring) free()
 		})
+		this add("inRange", func {
+			span := TimeSpan milliseconds(30)
+			expect(!span inRange(TimeSpan milliseconds(1), TimeSpan milliseconds(25)))
+			expect(!span inRange(TimeSpan milliseconds(31), TimeSpan milliseconds(50)))
```

Comment:
`expect(span inRange(TimeSpan milliseconds(31), TimeSpan milliseconds(50)), is false)` and similarly on the line above